### ISSUE TITLE
Update linux distro in github workflow file

### ISSUE THREAD

**Issue by afwillia** [Sage-Bionetworks/dccmonitor#126]
GitHub actions don't support version 16.04 https://github.com/Sage-Bionetworks/dccmonitor/settings/actions/runners

Unless you have a private runner, which I don't see
https://github.com/Sage-Bionetworks/dccmonitor/settings/actions/runners

**Comment by avanlinden** [comment]
Yeah no private runner, it's just very out of date and needs to be updated.
